content/realtime/channel-lifecycle.textile

```diff
@@ -0,0 +1,141 @@
+---
+title: Channel Metadata and Lifecycle Events
+section: realtime
+index: 15
+jump_to:
+  Help with:
+    - Overview#overview
+    - Metachannels#metachannels
+    - Lifecycle Events#lifecycle-events
+    - Reactor Rules#reactor
+    - Permissions#permissions
+  Channel API Types:
+    - ChannelDetails#channel-details
+    - ChannelStatus#channel-status
+    - Occupancy#occupancy
+---
+
+h2(#overview). Overview
+
+The realtime "channel lifecycle":#lifecycle-events and "occupancy":#occupancy events are published onto "metachannels":#metachannels that can be subscribed to using our existing "realtime libraries":/realtime. This is often used in conjunction with the "Ably Reactor":/reactor, which allows you to send this data to your "serverless endpoints":/general/functions, "queues":general/queues, and "streams":general/firehose.
+
+The "REST API":/rest-api can also be used "enumerate all active channels":/rest/channel-lifecycle#enumeration-rest, or obtain the status of an "individual channel":#metadata-rest. At present, REST "requests":/rest/usage#request for lifecycle events will not count towards your message limit. If you are making use of "Reactor":/reactor however, then each message sent over Reactor *will* count towards your message limits.
+
+h2(#use-cases). Use cases
+
+Having access to channel metadata can provide numerous benefits. In a scenario where the number of subscribers to a channel goes well beyond a hundred, usage of other options such as "presence":/core-features/presence become less effective. You could instead make use of our "occupancy events":#occupancy and "channel metadata":#metachannels to check the number of active subscribers, getting accurate values easily.
+
+Equally, you may want to only publish data when you know someone is subscribed for your data. Therefore, "channel lifecycle events":#lifecycle-events can notify you when a channel becomes active, and can notify you when the last subscriber leaves the channel.
```

Comment:
Perhaps:

'Equally, you may want to only publish data when you know someone is subscribed for your data.' -> ' Equally, you may want to publish your data only if there is a subscriber for that data" ?

Reply:
The channel occupancy/lifecycle events only notify about those numbers or events so perhaps saying that a user will be notified when the last subscriber leaves is a bit misleading.

Maybe just say, you can set up notifications for when the last subscriber leaves the channel, using the occupancy updates?